.github/workflows/build-test.yaml

```diff
@@ -0,0 +1,68 @@
+name: Build and Test
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    shell: bash
+
+env: 
+  HOMEBREW_NO_AUTO_UPDATE: 1
+
+jobs:
+
+  build:
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-18.04]
+        python-version: [3.8]
+        minimal: [false]
+        answer-tests: [false]
+        include:
+          - os: ubuntu-18.04
+            python-version: 3.6
+            minimal: true
+            answer-tests: false
+          - os: ubuntu-18.04
+            python-version: 3.7
+            minimal: false
+            answer-tests: true
+
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Install Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+        fetch-depth: 0
+    - uses: s-weigand/setup-conda@v1
+      if: matrix.os == 'windows-latest'
+      with:
+        update-conda: true
+        conda-channels: conda-forge
+        activate-conda: true
+        python-version: ${{matrix.python-version}}
+    - name: Show env
+      shell: bash
+      run: |
+        echo PATH $PATH
+        echo python `which python`
+        python -c "import sys; print(sys.version)"
+    - name: Install dependencies and yt
+      shell: bash
+      run: |
+        if [ "${{ matrix.minimal }}" == "true" ]; then
+          export MINIMAL=1
+        fi
+        source ./tests/ci_install.sh
+    - name: Run Tests
+      run: |
+        if [ "${{ matrix.answer-tests }}" == "true" ]; then
```

Comment:
I believe ti's possible to use an `env:` [block](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#env) where we set the variables to the values here, as described [in this SO post](https://stackoverflow.com/questions/59302035/switch-environment-variable-in-github-actions). That might make this a bit more terse.

Reply:
Done.